resources/js/beatmap-discussions/nominator.tsx

```diff
@@ -214,118 +228,84 @@ export class Nominator extends React.PureComponent<Props, State> {
         </div>
       </Modal>
     );
-  };
-
-  requiresFullNomination = (mode: GameMode) => {
-    let req;
-    let curr;
-
-    if (this.props.beatmapset.nominations?.legacy_mode) {
-      req = this.props.beatmapset.nominations?.required;
-      curr = this.props.beatmapset.nominations?.current;
-    } else {
-      req = this.props.beatmapset.nominations?.required[mode];
-      curr = this.props.beatmapset.nominations?.current[mode];
-    }
+  }
 
-    return (curr === (req ?? 0) - 1) && !this.hasFullNomination(mode);
-  };
+  private renderModalContentHybrid() {
+    if (this.playmodes == null) return null;
 
-  showNominationModal = () => this.setState({ visible: true });
+    return (
+      <>
+        {trans('beatmapsets.nominate.dialog.which_modes')}
+        <div ref={this.checkboxContainerRef} className={`${bn}__checkboxes`}>
+          {map(this.playmodes, (mode: GameMode) => {
```

Comment:
a normal map is fine, I think 🤔 

(also I've been calling playmode/ruleset ruleset for new codes)